dockers/docker-lldp/docker-lldp-init.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
+. /usr/bin/start.sh
```

Comment:
How to make sure this won't interffer with create_switch in the future?

Reply:
I can't think of a way to make sure this won't interfere in the future.
I think the best approach to improve the performance and ensure the stability of the solution is to delay these services in 90 seconds (fastboot criteria for control plane), this improve the performance even more.
The suggested services to delay is: dhcp_relay, radv, pmon and lldp.
In the future, as part of the HLD I am currently working, we will have a better way to indicate the end of fast-reboot flow and we can start the services according to it.
@liat-grozovik  @stepanblyschak @vaibhavhd @yxieca what do you think about this approach?

Reply:
> @shlomibitton request change indication was removed. Still need to handle, right?

Still waiting for  @vaibhavhd and @yxieca reply for the new suggestion following Stepan comment.
To make it stable we can use the suggested approach, need to get the approval first and I will update the PR accordingly.

Reply:
I am confused about what are we proposing. This PR is preempting lldp execution  (`LLDP consume the CPU before create_switch is executed and not interfering it.`), but we are also saying that in future PR we are going to defer the execution post fast-reboot completion.
Do we need this change right now, if we are soon going to depend upon fast-reboot end-indicator to start the deferred set of services?
I am not sure what is the impact of delaying LLDP.

Reply:
Ok, I understand the new-design in future-branch part.
Still some open questions. I agree that lldp startup sequence can be adjusted to prevent overlap with create-switch. But my doubt was - should it be done before create switch (this PR), or after create-switch (deferred start in 202205)?

Also I believe that this change will affect warm-boot process too. Do you agree? If yes, then title and description should be updated.

Reply:
I suggest to change this PR with the after create-switch approach - deferred start.
Following Stepan comment, this approach is lack of a sync mechanism to ensure it will not overlap again in the future.
The other suggestion is to delay the entire service startup until init flow has finished with a 90 seconds timer and this is much more reliable and a bit similar to the new design of 202205.
I mean, with the new implementation these services will be delayed as well, the only difference is to start them with a different mechanism and not a timer.
I agree it will affect warm-reboot and the description will be modified as well.
If this approach is acceptable I will update all as needed.

Reply:
@vaibhavhd what do you think? Can I proceed and modify the PR to this suggestion?

Reply:
I replied back on the email regarding this. Sorry for stalling this. The new suggested approach LGTM.
However, I am not confident about impact of delaying lldp with a 90s timer. May be @prsunny  or @yxieca can vet this from lldp point of view.

At this stage, I also would suggest to put two different PRs for pmon and lldp.

Reply:
thanks @vaibhavhd, I will create a PR and test it.

Reply:
@vaibhavhd @liat-grozovik @stepanblyschak @yxieca I created 2 new PR's with the suggested approach, can you please review? Thanks

PMON: https://github.com/Azure/sonic-buildimage/pull/10567
LLDP: https://github.com/Azure/sonic-buildimage/pull/10568